.circleci/conditional_config.yml

```diff
@@ -385,7 +392,7 @@ jobs:
 
   next_rails-rspec:
     docker:
-      - <<: *frontend_base
+      - <<: *frontend_ci
```

Comment:
we don't need chromium here

Reply:
We run the feature specs in this job, so Chromium is needed.

Reply:
fixed.